apiserver/pkg/util/job.go

```diff
@@ -0,0 +1,63 @@
+package util
+
+import (
+	api "github.com/ray-project/kuberay/proto/go_client"
+	rayalphaapi "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RayJob struct {
+	*rayalphaapi.RayJob
+}
+
+const rayJobDefaultVersion = "1.13"
+
+// NewRayJob creates a RayJob.
+func NewRayJob(apiJob *api.RayJob, computeTemplateMap map[string]*api.ComputeTemplate) *RayJob {
+	var clusterSpec rayalphaapi.RayClusterSpec
+
+	if apiJob.ClusterSpec != nil && len(apiJob.ClusterSelector) == 0 {
+		clusterSpec = *buildRayClusterSpec(rayJobDefaultVersion, apiJob.ClusterSpec, computeTemplateMap)
+	}
+
+	rayJob := &rayalphaapi.RayJob{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        apiJob.Name,
+			Namespace:   apiJob.Namespace,
+			Labels:      buildRayJobLabels(apiJob),
+			Annotations: buildRayJobAnnotations(apiJob),
+		},
+		Spec: rayalphaapi.RayJobSpec{
```

Comment:
We need to expose TTL setting as well. Check latest job definition

Reply:
added ttl in job proto